plugin.xml

```diff
@@ -52,31 +52,14 @@
         <config-file target="AndroidManifest.xml" parent="/manifest">
             <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
         </config-file>
-
-        <config-file target="AndroidManifest.xml" parent="application">
-            <provider
-                android:name="com.imagepicker.plus.ImagePickerProvider"
-                android:authorities="${applicationId}.imagepickerprovider"
-                android:exported="false"
-                android:grantUriPermissions="true">
-                <meta-data
-                    android:name="android.support.FILE_PROVIDER_PATHS"
-                    android:resource="@xml/imagepicker_provider_paths" />
-            </provider>
-        </config-file>
   
-        <framework src="androidx.activity:activity:1.3.1" />
-        <framework src="androidx.fragment:fragment:1.3.6" />
         <framework src="androidx.exifinterface:exifinterface:1.3.3" />
 
         <!-- Enable data binding for tedimagepicker -->
-        <framework src="io.github.ParkSangGwon:tedimagepicker:1.2.8" />
+        <framework src="io.github.ParkSangGwon:tedimagepicker:1.2.7" />
```

Comment:
Is there a reason for the package downgrade?

Reply:
Tedimagepicker latest version (1.2.8) conflict with cordova plugin.
So we must revert back to 1.2.7 to fixed it.